lib/data_update_scripts/20201228194641_append_collective_noun_to_community_name.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class AppendCollectiveNounToCommunityName
+    def run
+      return "#{SiteConfig.community_name} #{SiteConfig.collective_noun}" unless SiteConfig.collective_noun_disabled?
```

Comment:
@juliannatetreault This doesn't actually _update_ the SiteConfig at all, it just returns a string. I think that's probably why the test isn't doing what you expect/want it to -- you need actually update the `SiteConfig.community_name` similar to what you [previously did](https://github.com/forem/forem/pull/11846/files#diff-620cde66dafdd0fed658093095e053565f368bee979f03ba83bc2eb9a2fe9dabR7) in your initial PR

Reply:
I was totally overthinking this yesterday and looking at it with fresh eyes this morning made me 🤦‍♀️  --  thank you for the feedback!